pkg/controller/controller_instance.go

```diff
@@ -325,7 +319,7 @@ func (c *controller) reconcileServiceInstance(instance *v1beta1.ServiceInstance)
 // ReconciledGeneration for status API migration.
 // Returns true if the status was updated (i.e. the iteration has finished and no
 // more processing needed).
-func (c *controller) initObservedGeneration(instance *v1beta1.ServiceInstance) (bool, error) {
+func (c *controller) initObservedGeneration(instance *v1beta1.ServiceInstance) (*v1beta1.ServiceInstance, error) {
```

Comment:
Please update comments which indicate this function returns true if the status was updated